.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @paketo-buildpacks/stacks-maintainers
```

Comment:
I think this needs to be `@paketo-community/stacks-maintainers`

Reply:
@robdimsdale thanks, fixed.